### PR TITLE
Merge Card Component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created the following cards:
1. Horizontal Card
2. Vertical card with badge
3. Vertical card with text overlay
4. Vertical card with dismiss
5. Vertical card without footer

Card

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  position: relative;
+  background-color: var(--color-white);
+  width: 350px;
+  height: max-content;
+  border-radius: var(--radius-extra-large);
+  box-shadow: var(--box-shadow);
+  overflow: hidden;
+}
+
+.card__img {
```
Comment (shivampandey0) [inline components/card/card.css]: why __ here? all other classes mostly use -

**Review by shivampandey0 — COMMENTED** [review]
Code looks good with extensive use of CSS variables.
- Added a Suggestion to keep the naming pattern consistent.

File: components/card/card.css (hunk 1 above)
Comment (Hamdrive) [reply components/card/card.css]: Thank you for catching this! 
Yes it was an old class that I forgot to clean up before pushing

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

**Comment by Hamdrive** [comment]
Changes made based on review. Merging to dev

**Pull request merged**